CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# CHANGES in AVA 3
+
+## 3.0.0
```

Comment:
Do we need a tag@alpha?

Reply:
it's a todo, we can modify it before publish